huaweicloud/services/rds/resource_huaweicloud_rds_instance.go

```diff
@@ -288,6 +288,16 @@ func ResourceRdsInstance() *schema.Resource {
 				Computed: true,
 			},
 
+			"maintain_begin": {
```

Comment:
Do `maintain_begin` and `maintain_end` have default values? If yes, please add Computed

Reply:
done